buildings_tools/src/buildings_tools/BuildingSizeDialog.java

```diff
@@ -21,11 +23,25 @@
     private final JCheckBox caddr = new JCheckBox(tr("Use Address dialog"));
     private final JCheckBox cAutoSelect = new JCheckBox(tr("Auto-select building"));
     private final JCheckBox cAddrNode = new JCheckBox(tr("Use address nodes under buildings"));
+    private final JRadioButton circleRadio = new JRadioButton("Circle");
+    private final JRadioButton rectangleRadio = new JRadioButton("Rectangle");
 
     public BuildingSizeDialog() {
-        super(tr("Set buildings size"));
+        super(tr("Set buildings size and shape"));
 
-        addLabelled(tr("Buildings width:"), twidth);
+        circleRadio.setToolTipText("Press SHIFT + 'C' when drawing building");
+        rectangleRadio.setToolTipText("Press SHIFT + 'R' when drawing building");
```

Comment:
Should use the internationalization `tr()` method here.  Need to research how text translations are added for new terms.

Reply:
Translations are added to Launchpad every night (3.30am CET) by scanning source files from OSM SVN and @JOSM Github repos. Then everyone can translate them directly on Launchpad. At the end of the month, we manually update JOSM core translations. For plugins, this is not always done every month. See https://josm.openstreetmap.de/wiki/Translations#Softwaretranslation for more details. Also, you shouldn't hardcode the shortcut in the translated string but use a parameterized string, as shortcuts can be changed by end-users.